README.md

```diff
@@ -129,60 +117,118 @@ requests please open an issue.
-
-## Processing during build
+Our ambition is to remain pragmatic while selecting semver bump types.
+Technically even a patch change can introduce breaking changes to some clients
+downstream. Use the table below as a guide only.
+
+| Bump type | Discussion                                                                                                                                                                                                                                               |
+| --------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Major     | Implies the addition of major changes that may be backwards incompatible, e.g. the transition of a property from optional to required. The schemas will fail to validate Sketch documents considered valid by the previous version before the major bump |
+| Minor     | Implies the addition of a new backwards compatible feature, e.g. the addition of a new optional property                                                                                                                                                 |
+| Patch     | Implies a bug fix or trivial change, but could introduce a breaking change if a dependant package was previously implementing buggy schemas                                                                                                              |
+| Pre       | Bumps between prereleases convey no specific semantics                                                                                                                                                                                                   |
+
+### Workflows
+
+#### Conventional commits
+
+Try and use the [conventional commits](https://www.conventionalcommits.org/)
+convention when writing commit messages. This isn't enforced, but you can use
+the `yarn commit` command (in place of `git commit -m "foo"`) to open an
+interactive CLI to walk you through generating a properly formatted commit
+message.
+
+#### Updating the schemas
+
+1. Update the schema yaml source files to reflect your changes to the
+   specification
+1. Use the `yarn validate-schemas` script to check that your changes are valid
+   according to JSON Schema Draft 7
+1. Use the `yarn validate-file` and `yarn validate-reference-files` scripts to
+   validate real Sketch files with your updated schemas
+1. Determine the semver bump type and call `yarn changeset` to create an intent
+   to release your changes (read more about changesets
+   [here](https://github.com/atlassian/changesets)).
+1. Open a PR to `master`
+
+#### Creating a release
+
+1. Merge the release PR maintained by the changesets
+   [GitHub action](https://github.com/changesets/action).
+
```

Comment:
I wonder if it would be better to flip the order:

- Pre-release
- Release

even if the former is still work in progress. Also, perhaps we can drop the the _Creating a_?